src/tests/java/picard/fingerprint/FingerprintCheckerTest.java

```diff
@@ -26,4 +36,45 @@ public void testRandomSublist() throws Exception {
 
         Assert.assertEquals(FingerprintChecker.randomSublist(list, 2).size(), 2);
     }
+
+    @DataProvider(name="pLoH")
+    public Iterator<Object[]> pLohData(){
+        List<Object[]> listOfDoubles = new ArrayList<>();
+
+        for(int i=1;i<20; i++) {
+            listOfDoubles.add(new Object[]{i/40D});
+        }
+        return listOfDoubles.iterator();
+    }
+
+    @Test(dataProvider = "pLoH")
+    public void testMatchResults(final double pLoH) {
```

Comment:
new test. checks that if we test a hom fp against a het fp, it only give good results if the het is "normal" and the hom is "tumor"